dialect/StablehloOps.cpp

```diff
@@ -5667,6 +5569,163 @@ LogicalResult UniformDequantizeOp::inferReturnTypeComponents(
   return success();
 }
 
+//===----------------------------------------------------------------------===//
+// Assembly - Custom Type Directives
+//===----------------------------------------------------------------------===//
+
+// Declarative `custom<SameOperandsAndResultType>(...)` implementation:
+// Pretty print for ops with many operands, but one result type, simplifies
+// print if all operand types match the result type. Based on `printOneResultOp`
+// and `parseOneResultSameOperandTypeOp` from tfl_ops.cc and SPIRVOps.cpp.
+//
+// Example:
+//   custom<SameOperandsAndResultType>(type($result), type($operand1),
+//   type($operand2))
```

Comment:
When going over this PR again, I noticed that this comment still uses the old result-then-operands order. Can you fix in a follow-up PR?